circle.yml

```diff
@@ -171,14 +171,19 @@ jobs:
       - run:
           name: "Generate documentation"
           command: doxygen Doxyfile
+      - add_ssh_keys:
+          fingerprints: [ 57:0c:50:3c:a3:72:4c:a9:28:e8:77:44:87:3b:a2:9a ]
       - run:
           name: "Upload documentation"
           command: |
+            # Remove problematic symbolic link
+            rm bindings/rust/evmc-sys/evmc.h
```

Comment:
What is the symptom? I guess this wasn't cause any issues because it is related to Rust only.

Reply:
Jekyll build fails. I didn't investigated further.